### DISCUSSION TITLE
Is there, will there, be a Rails SDK for SQLite Cloud?

### DISCUSSION THREAD

**Discussion by ian-bradbury** [sqlitecloud/discussions#15] [Feature request] [answered]
The technology sounds amazing and I am particularly interested in a Rails interface to the cloud platform.  SQLite Cloud + Rails would help me to target all the platforms.

**Comment by jacobprall** [comment, marked as answer]
Hey Ian! Thank you for the request - we are currently building out integrations, and Rails is high on our list. We'll keep you posted!
